redbeams-api/src/main/java/com/sequenceiq/redbeams/api/endpoint/v4/database/responses/DatabaseV4Response.java

```diff
@@ -0,0 +1,93 @@
+package com.sequenceiq.redbeams.api.endpoint.v4.database.responses;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonInclude.Include;
+import com.sequenceiq.cloudbreak.api.endpoint.v4.common.responses.SecretV4Response;
+import com.sequenceiq.cloudbreak.doc.ModelDescriptions.Database;
+import com.sequenceiq.redbeams.api.endpoint.v4.database.base.DatabaseV4Base;
+import com.sequenceiq.redbeams.doc.ModelDescriptions;
+
+import io.swagger.annotations.ApiModel;
+import io.swagger.annotations.ApiModelProperty;
+
+@ApiModel
+@JsonInclude(Include.NON_NULL)
+public class DatabaseV4Response extends DatabaseV4Base {
+
+    @ApiModelProperty(ModelDescriptions.CRN)
+    private String crn;
+
+    @ApiModelProperty(ModelDescriptions.CREATED)
+    private Long creationDate;
+
+    @ApiModelProperty(value = Database.DB_ENGINE, required = true)
+    private String databaseEngine;
+
+    @ApiModelProperty(value = Database.CONNECTION_DRIVER_NAME, required = true)
+    private String connectionDriver;
+
+    @ApiModelProperty(value = Database.DB_ENGINE_DISPLAYNAME, required = true)
+    private String databaseEngineDisplayName;
+
+    @ApiModelProperty(Database.USERNAME)
+    private SecretV4Response connectionUserName;
```

Comment:
I switched the database server code over to use the secret-engine module, which replaces this with SecretResponse. It's a good idea to do the same swap here as well.

Reply:
Yep, thank you! Done.